### DISCUSSION TITLE
小白关于新出卡显示的建议

### DISCUSSION THREAD

**Discussion by yuyuyuka** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#49] [Q&A]
大佬我又来了，透视工具非常强大很便利，但是每次出新卡和新马一旦带上，那就会变成一堆字母。当然我知道是因为事件文件还没更新，但是我想建议一下，能不能在不显示新卡/新事件的情况下显示正常的卡，这样好歹能带上，不然的话面对一堆错误警告，实在很难受，谢谢！

**Comment by EtherealAO** [comment]
不更新数据怎么能知道新卡叫什么呢。
目前并没有直接读取master.mdb而不是使用解析后数据的计划，因为如果在打开master.mdb的情况下，游戏似乎是不能正常启动的
